sample/src/main/java/com/readystatesoftware/chuck/sample/MainActivity.kt

```diff
@@ -0,0 +1,99 @@
+package com.readystatesoftware.chuck.sample
+
+import android.content.Context
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.view.View
+import android.widget.Button
+import com.readystatesoftware.chuck.api.Chuck
+import com.readystatesoftware.chuck.api.ChuckCollector
+import com.readystatesoftware.chuck.api.ChuckInterceptor
+import com.readystatesoftware.chuck.api.RetentionManager
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class MainActivity : AppCompatActivity() {
+
+    private var collector: ChuckCollector? = null
```

Comment:
You may use a `lateinit` here, collector is created in the onCreate, and is not reset to null ever.